### PR TITLE
Add comparison page

### PR DESCRIPTION
**Pull request by R0ixy**
https://app.asana.com/0/1202525679663009/1202943623164029/f

image

### CODE REVIEW

File: frontend/src/components/collapse-component/collapse-element/styles.module.scss [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 .show {
   height: auto;
-  max-height: 1500px;
+  max-height: 2000px;
   transition-duration: 0.8s;
 }
```
Comment (oleksandr-danylchenko) [inline frontend/src/components/collapse-component/collapse-element/styles.module.scss:20]: Why `2000px`?

File: frontend/src/components/comparison-page/comparison-page.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            <CompTopTableBar />
+            <GeneralComparisonTable />
+            <OptionsSubtable title="design" />
+            <OptionsSubtable title="sound" />
+            <OptionsSubtable title="security" />
+            <OptionsSubtable title="multimedia" />
+            <OptionsSubtable title="comfort" />
+            <OptionsSubtable title="optics" />
+            <OptionsSubtable title="auxiliary" />
+            <OptionsSubtable title="upholstery" />
```
Comment (oleksandr-danylchenko) [inline frontend/src/components/comparison-page/comparison-page.tsx]: I believe that it can be done more elegantly using the array of options and `.map`

**Review by oleksandr-danylchenko — APPROVED** [review]

File: frontend/src/components/collapse-component/collapse-element/styles.module.scss (hunk 1 above)
Comment (R0ixy) [reply frontend/src/components/collapse-component/collapse-element/styles.module.scss:20]: We have dynamic tables height and 1500px was not enough on some screen sizes

Comment (oleksandr-danylchenko) [reply frontend/src/components/collapse-component/collapse-element/styles.module.scss:20]: Okay. Why do we need to have `max-height` at all?

**Review by oleksandr-danylchenko — APPROVED** [review]

File: frontend/src/components/collapse-component/collapse-element/styles.module.scss (hunk 1 above)
Comment (R0ixy) [reply frontend/src/components/collapse-component/collapse-element/styles.module.scss:20]: That's an interesting question, but collapse component don't work without it

**Pull request merged**